packages/grafana-ui/src/components/VizTooltip/models.gen.ts

```diff
@@ -4,3 +4,7 @@
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 export type TooltipMode = 'single' | 'multi' | 'none';
```

Comment:
Maybe let's rename it, similar to LegendDisplatyMode, to `TooltipDisplayMode` and use enum instead.